.buildkite/pipeline.e2e.yml

```diff
@@ -0,0 +1,18 @@
+steps:
+  - label: "e2e test:desktop"
+    plugins:
+      - docker-compose#v3.5.0:
+          run: e2e
+          env:
+            - PLAYWRIGHT_BASE_URL
+            - USE_STAGE_APIS
```

Comment:
Where does this get set?

Reply:
This specifies that these env vars are settable - when you trigger the build from another pipeline you add the env vars to that. https://buildkite.com/docs/pipelines/trigger-step#trigger-step-attributes

Reply:
👍